package.json

```diff
@@ -48,7 +48,7 @@
     "zone.js": "0.8.14"
   },
   "devDependencies": {
-    "@angular/cli": "1.6.0",
+    "@angular/cli": "1.6.5",
```

Comment:
CLI update is another PR too. Could you remove it from the other one.

Reply:
@sumitarora Just to be sure: I understood this comment correctly to leave the CLI update in _this_ PR but modify the _other_ ones so they don't include it anymore right?